### PR TITLE
3265 include integration test to circleci

### PR DESCRIPTION
**Pull request by melissapalmer**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -73,6 +74,44 @@ jobs:
...
+      - run: npm install -g newman
+      - run: newman run -h
+      # Add the required hostname entries for the integration tests.
+      - run: echo 127.0.0.1 a3s-identity-server | sudo tee -a /etc/hosts
+      # Checkout the repository
+      - checkout
+      # # Build the two required docker images into the local docker image repo from this version of the code.
+      - run: docker build -t grindrodbank/dokuti:latest -f ~/repo/Dockerfile ~/repo
+      # # Start the quickstart instance, as it models a fully featured running A3S environment, which is ideal for running Postman tests against.
+      - run: docker-compose -f ~/repo/quickstart/docker-compose/docker-compose.yml -f ~/repo/quickstart/docker-compose/docker-compose.override.yml up -d
```
Comment (VictorSzewczenko) [inline .circleci/config.yml:97]: I hope this trick eased your ordering pains?

File: .circleci/config.yml [hunk 2]
```diff
@@ -73,6 +74,44 @@ jobs:
...
+      - run: sleep 30
+      - run: docker ps --all 
+      # # Run the Newman to execute the collection against the quickstart environment.
+      # the sed command below relates to an error we are getting with newman `Unknown Encoding: latin1` it will convert the file to UTF-8 w/BOM.
+      # see more details at: 
+        # https://github.com/postmanlabs/newman/issues/1959
+        # https://stackoverflow.com/questions/54900481/newman-unknown-encoding-latin1-pop-up-when-running-newman-cli-on-aws-codebuild
+      - run: | 
+          cd ~/repo/postman/
+          sed -i '1s/^/\xef\xbb\xbf/' Dokuti.postman_collection.json
```
Comment (VictorSzewczenko) [inline .circleci/config.yml:110]: This UTF BOM encoding step is the BOM! Thanks for this tip, I have been doing this manually till now.

File: postman/test-file.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+test-file.txt for postman automation scripts
```
Comment (VictorSzewczenko) [inline postman/test-file.txt:1]: Not sure if these should be here?

File: .circleci/config.yml (hunk 1 above)
Comment (melissapalmer) [reply .circleci/config.yml:97]: No this does not help with ordering pains, it’s used along with docket-compose override functionality such that in development you don’t start up the application you are Working on (that can be started in the ide). But it still keeps quick start with simple 
docket-compose up -d 
intact.
(Which by default picks up both of the above files) 

 https://docs.docker.com/compose/extends/

File: postman/test-file.txt (hunk 3 above)
Comment (melissapalmer) [reply postman/test-file.txt:1]: Some of the postman cases use this file to upload to Dokuti service. Therefore yes it’s needed. 

Am I missing something not sure why you think it’s wrong?

Comment (VictorSzewczenko) [reply postman/test-file.txt:1]: I just reviewed late and forgot that those files are totally needed. :)

**Pull request merged**